exporter/src/main/as/flump/export/EditFormatsWindow.mxml

```diff
@@ -41,20 +41,6 @@
         <s:GridColumn dataField="scaleFactorsString" headerText="Scale Factors"/>
         <s:GridColumn dataField="textureBorder" headerText="Texture Border"/>
         <s:GridColumn dataField="maxAtlasSize" headerText="Max Atlas Size"/>
-        <s:GridColumn dataField="optimize" headerText="Optimize">
-          <s:itemEditor>
-            <fx:Component>
-              <s:ComboBoxGridItemEditor>
-                <s:dataProvider>
-                  <s:ArrayList>
-                    <fx:String>Memory</fx:String>
-                    <fx:String>Speed</fx:String>
-                  </s:ArrayList>
-                </s:dataProvider>
-              </s:ComboBoxGridItemEditor>
-            </fx:Component>
-          </s:itemEditor>
-        </s:GridColumn>
```

Comment:
Lots of small textures atlases generate lots of draw calls which kills the performance. For example we have a (quite complex) character for which Flump generated 2 atlases and this resulted in 50+ draw calls  per frame during an animation.

Memory should not be an issue if we use compressed textures (.atf) .atf files are stored compressed in the GPU memory thus if they have big blank regions does not matter. But as I wrote above since MaxRects allows to fit more stuff into a single texture the number of draw calls will decrease leading to better performance.